tigerpath/majors_and_certificates/majors/ART_2018.json

```diff
@@ -0,0 +1,272 @@
+      "name": "Michael Koortbojian",
+      "email": "romanist@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Nathan A. Arrington",
+      "email": "nta@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Track",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": null,
+      "req_list": [
+        {
+          "name": "History of Art",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "The History of Art Track explores the history of art in connection with a general program of humanistic education. It demands broad exposure to the visual arts produced in different time periods and cultures and trains majors to understand works of art as intimately related to the historical, intellectual, social, and political fabrics of which they were a part. Students in the History of Art Track are encouraged to explore a range of media including architecture, painting, sculpture, photography, film, and new media.",
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 2,
+              "explanation": "Any two courses offered by the Department of Art and Archaeology.",
+              "double_counting_allowed": true,
+              "course_list": [
+                "ART *"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "A total of 10 courses in the Department of Art and Archaeology, including ART 100, ART 400 (Junior Seminar) and two seminars at the 400- or 500-level. Students must also take at least one course in each of the following three areas: Group 1 (ancient), Group 2 (medieval/early modern), and Group 3 (modern/contemporary). Thematic courses spanning more than one area will be allocated to a distribution area on a case-by-case basis. ART 100, ART 400, and ART 401 count as departmentals but not as distribution courses. In choosing courses to satisfy the distribution requirement, students are encouraged to explore a range of media, e.g., architecture, painting, sculpture, photography, works on paper, film. 7 of the 10 courses need to be taught by Art and Archaeology faculty.",
+              "req_list": [
+                {
+                  "name": "Required Courses",
+                  "max_counted": 1,
+                  "min_needed": "ALL",
+                  "explanation": null,
+                  "course_list": [
+                    "ART 100"
+                  ]
+                },
+                {
+                  "name": "Seminars",
+                  "max_counted": 1,
+                  "min_needed": 2,
+                  "explanation": "<i>Editor's Note: Many of these courses fall under this requirement but as TigerPath does not verify this.</i>\n\nTwo seminars at the 400- or 500-level",

```

Comment:
It seems like this same editor's not has been copied to multiple different locations, all with an extra "as" included!